src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/controller/ReferralController.kt

```diff
@@ -142,9 +143,29 @@ class ReferralController(
     @Nullable @RequestParam(name = "unassigned", required = false) unassigned: Boolean?,
     @Nullable @RequestParam(name = "assignedTo", required = false) assignedToUserId: String?,
     @PageableDefault(page = 0, size = 50, sort = ["sentAt"]) page: Pageable,
-  ): Page<SentReferralSummaryDTO> {
+  ): Page<SentReferralSummaryForDashboardDTO> {
     val user = userMapper.fromToken(authentication)
-    return (referralService.getSentReferralsForUser(user, concluded, cancelled, unassigned, assignedToUserId, page) as Page<Referral>).map { SentReferralSummaryDTO.from(it) }.also {
+    return (referralService.getSentReferralsForUser(user, concluded, cancelled, unassigned, assignedToUserId, page) as Page<ReferralForDashboard>).map { SentReferralSummaryForDashboardDTO.from(it) }.also {
+      telemetryClient.trackEvent(
+        "PagedDashboardRequest",
+        null,
+        mutableMapOf("totalNumberOfReferrals" to it.totalElements.toDouble(), "pageSize" to page.pageSize.toDouble())
+      )
+    }
+  }
+
+  @JsonView(Views.SentReferral::class)
```

Comment:
i don't think we need the JsonView annotation on this method. 

This annotation will only affect responses which contain DraftReferralDTOs.

if you think it's better safe than sorry, we can include this here too. i don't mind.